src/hooks/useStreamChat.ts

```diff
@@ -70,13 +70,15 @@ export function useStreamChat({
       redo,
       attachments,
       selectedComponent,
```

Comment:
same as elsewhere- let's remove `selectedComponent` and just use `selectedComponents`